python/llm/example/GPU/Pipeline-Parallel-FastAPI/run.sh

```diff
@@ -0,0 +1,13 @@
+source /opt/intel/oneapi/setvars.sh
+export no_proxy=localhost
+export FI_PROVIDER=tcp
+export OMP_NUM_THREADS=8
+
+export USE_XETLA=OFF
+export SYCL_PI_LEVEL_ZERO_USE_IMMEDIATE_COMMANDLISTS=2
+export TORCH_LLM_ALLREDUCE=0
+export CCL_DG2_ALLREDUCE=1 # For internal CCL
+
+export MODEL_PATH=1
+CCL_ROOT=./1ccl_dg2_allreduce_20240308  LD_LIBRARY_PATH=./1ccl_dg2_allreduce_20240308/src:./1ccl_dg2_allreduce_20240308/deps/mpi/lib:/opt/intel/oneapi/2024.0/lib \
```

Comment:
These are not public available packages right?

Reply:
Yes, will remove